.circleci/config.yml

```diff
@@ -217,6 +217,19 @@ jobs:
             which javac && javac -version
             SKIP_FORMAT_BUCK_CHECKS=1 PRINT_PARALLEL_OUTPUTS=1 make V=1 J=32 -j32 rocksdbjava jtest
 
+  build-examples:
+    machine:
+      image: ubuntu-1604:201903-01
+    resource_class: 2xlarge
```

Comment:
2xlarge is a waste we only do "-j4".

Reply:
ohkk. I will change it then to medium then. Thanks